load_questions.dart

```diff
@@ -35,7 +38,7 @@ void main() async {
 
   for (final question in questions) {
     await db.createDocument(
-      documentId: "unique()",
+        documentId: "unique()",
         collectionId: collectionId,
         data: question,
         read: ['*'],
```

Comment:
Would you please also update this read permission as `*` is deprecated?

Reply:
Okay.